llvm/lib/Target/AIE/AIEPostPipeliner.h

```diff
@@ -43,14 +44,65 @@ class NodeInfo {
   int ModuloCycle = 0;
   // Cycle / II
   int Stage = 0;
+
+  int StaticEarliest = 0;
+  int StaticLatest = -1;
```

Comment:
Can you document those? In particular what "static" means in that context. I.e. is it invariant in the strategy and the II? Or one of them?